{"author":{"id":"7fc4f5c20095d2b147a39f50b9986d3b113b75ea9994098e8b4de607b0f3fd8e"},"ops":[{"type":1,"timestamp":1617543675,"metadata":{"github-id":"MDU6SXNzdWU4NDk4NzczNzc=","github-url":"https://github.com/MichaelMure/git-bug/issues/615","origin":"github"},"nonce":"ABIQWTa6RBdnqeehTplKUJWDZug=","title":"Dependabot can't resolve your Go dependency files","message":"Dependabot can't resolve your Go dependency files.\n\nAs a result, Dependabot couldn't update your dependencies.\n\nThe error Dependabot encountered was:\n\n```\ngithub.com/MichaelMure/git-bug/commands: cannot find module providing package github.com/MichaelMure/git-bug/commands\n```\n\nIf you think the above is an error on Dependabot's side please don't hesitate to get in touch - we'll do whatever we can to fix it.\n\n[View the update logs](https://app.dependabot.com/accounts/MichaelMure/update-logs/75226522).","files":null}]}